tests/data/site_root/ig.ini

```diff
@@ -0,0 +1,13 @@
+[IG]
+ig = input/ImplementationGuide-KidsFirst.json
+template = fhir.base.template
+usage-stats-opt-out = false
+copyrightyear = 2019+
```

Comment:
`copyrightyear` mismatches with `tests/data/site_root/input/ImplementationGuide-KidsFirst.json` (2019+ vs. 2020+)

Reply:
Hmm idk how that happened